README.md

```diff
@@ -101,7 +105,10 @@ const token = await getToken();
 
 ## Running in an Adobe I/O Runtime action
 
-The AIO IMS Library can also be used in an Adobe I/O Runtime action. In this case the IMS configuration must be set beforehand. The library is relying on the [Cloud State Library](https://github.com/adobe/aio-lib-state) to persist the access tokens across action invocations and reduce the number of requests to IMS.
+**Note that Project Firefly applications should not own the responsibility to generate their own IMS access tokens.
```

Comment:
@shaz also added a warning here suggested by @meryllblanchet , let me know if you want me to add this as part of another PR

Reply:
this is fine